Makefile

```diff
@@ -1,22 +0,0 @@
-DOCKER=docker
```

Comment:
I would like to keep the Makefile, even if it is not used for CI. It helps (me) getting stuff done quick. But if you don't think it is necessary I can keep it locally and exclude it from the repo.